tests/test_output.py

```diff
@@ -110,3 +110,31 @@ def test_io(self):
             np.testing.assert_almost_equal(
                 o3["interpolation_xgrid"], d["interpolation_xgrid"]
             )
+    
+    def test_tar_output(self, tmp_path):
+        out, obs = self.fake_output()
+        #dump fake output
+        o1 = output.Output(out)
+        temp_path = pathlib.Path(tmp_path)
+        fn = pathlib.Path("test.tar")
+        actual_path = pathlib.PurePath(temp_path, fn)
```

Comment:
Why do you need a `PurePath`?

Reply:
Even more: since `fn` you're using only once, you can just do:
```py
actual_path = temp_path / "test.tar"
```
and that's it :)

Reply:
Yes, you're right. I fixed it.